redhat/overlays/Dockerfile

```diff
@@ -16,8 +18,13 @@ LABEL io.openshift.tags="gitsign trusted-artifact-signer"
 LABEL summary="Provides the gitsign CLI binary for signing and verifying container images."
 LABEL com.redhat.component="gitsign"
 
-COPY --from=build-env /gitsign/gitsign /usr/local/bin/gitsign
-RUN chown root:0 /usr/local/bin/gitsign && chmod g+wx /usr/local/bin/gitsign
+COPY --from=build-env /gitsign/gitsign_cli_darwin_amd64 /usr/local/bin/gitsign_cli_darwin_amd64
+COPY --from=build-env /gitsign/gitsign_cli_linux_amd64 /usr/local/bin/gitsign_cli_linux_amd64
+COPY --from=build-env /gitsign/gitsign_cli_windows_amd64.exe /usr/local/bin/gitsign_cli_windows_amd64.exe
```

Comment:
It might be a good test to gzip these binaries and see how much smaller the image would be.